tests/xharness/XmlResultParser.cs

```diff
@@ -640,7 +640,8 @@ static void GenerateNUnitV3Failure (XmlWriter writer, string title, string messa
 				("inconclusive", "0"),
 				("skipped", "0"),
 				("asserts", "1"),
-				("date", XmlConvert.ToString (DateTime.Now, "yyyy-MM-dd"))
+				("run-date", XmlConvert.ToString (DateTime.Now, "yyyy-MM-dd")),
+				("start-time", DateTime.Now.ToString ("HH:mm:ss"))
```

Comment:
minor: you can (bots are good at this) have a `run-date of `X` for `X @ 23:59:59` and a a very early (past) `start-time`, e.g. `00:00:00`, for `X+1 @ 00:00:00`. Depending on how the data is gathered/analyzed this could generate illogical data. Using the same instance of `DateTime.Now` ensure both would be in sync.

Reply:
will do.